x-pack/plugins/maps/public/components/layer_addpanel/view.js

```diff
@@ -20,6 +20,7 @@ import {
   EuiFlyoutBody,
   EuiFlyoutFooter,
 } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
```

Comment:
Forgot to i18n the following
* `Choose data source` header from `_renderSourceSelect`
* `Cancel` button text